src/react-table.d.ts

```diff
@@ -0,0 +1,30 @@
+﻿/* eslint-disable @typescript-eslint/no-unused-vars */
+import "@tanstack/react-table";
+import { RowData } from "@tanstack/react-table";
+import { CSSProperties } from "react";
+
+interface DropdownColumnFilterOption {
+  label: string;
+  value: string | number;
+  disabled?: boolean;
+}
+
+interface DropdownColumnFilter {
+  options: DropdownColumnFilterOption[];
+}
+
+export interface EnumValue {
```

Comment:
@manni497 i cannot find that this is still used, do you know if we use that somewhere? else i will remove it.

Reply:
@neotob Yes, it's not used :D

Reply:
removed thanks :)